OWML.ModHelper.Menus/ModSelectorInput.cs

```diff
@@ -14,11 +14,17 @@ public class ModSelectorInput : ModInput<string>, IModSelectorInput
         private readonly OptionsSelectorElement _element;
         private int _count;
         private List<string> _options = new List<string>();
+        private Dictionary<string, string> _converter;
+        private Dictionary<string, string> _inverseConverter;
```

Comment:
Could probably use the same dictionary to store both "list" and "dictionary" alternatives?
In the list case, the dictionary could have the same item for both keys and values, or null values.